src/model_signing/_signing/sign_sigstore.py

```diff
@@ -107,9 +108,18 @@ def __init__(
               identity to the OIDC provider. If not provided, it is assumed
               that the client is public or the provider does not require a
               secret.
+            trust_config: A path to a custom trust configuration. When
+              provided, the signature verification process will rely on the
+              supplied PKI and trust configurations, instead of the default
+              Sigstore setup. If not specified, the default Sigstore
+              configuration is used.
         """
         if use_staging:
             trust_config = sigstore_models.ClientTrustConfig.staging()
+        elif trust_config:
```

Comment:
```suggestion
        elif trust_config is not None:
```

This is to make sure we don't confuse between a boleean and a None

Reply:
Although, I see that we didn't follow this in the rest of the code, so let's do in separate PR